storage/innobase/handler/handler0alter.cc

```diff
@@ -621,6 +621,16 @@ inline bool dict_table_t::instant_column(const dict_table_t& table,
 	}
 
 	dict_index_t* index = dict_table_get_first_index(this);
+	if (instant)
+	{
+	  instant->field_map= static_cast<field_map_element_t*>(
+            mem_heap_dup(heap, instant->field_map,
+                         (index->n_fields - index->first_user_field()) *
+                         sizeof *instant->field_map));
+	  instant= static_cast<dict_instant_t*>(
+            mem_heap_dup(heap, instant, sizeof *instant));
+	}
```

Comment:
This function uses the K&R style and TAB based formatting. Please follow that formatting here:
```c++
	dict_index_t* index = dict_table_get_first_index(this);
	if (instant) {
		instant->field_map= static_cast<field_map_element_t*>(
			mem_heap_dup(heap, instant->field_map,
```